xml/dm_registers.xml

```diff
@@ -626,7 +651,7 @@
         \begin{steps}{Writes to this register:}
             \item If the bus master is busy then accesses set \Fsberror, and
             don't do anything else.
-            \item Start a bus write of {\tt sbdata} to {\tt sbaddress}.
+            \item Start a bus write of the new value of {\tt sbdata} to {\tt sbaddress}.
             \item If \Fsbautoincrement is set, increment {\tt sbaddress}.
```

Comment:
Should we add "sets `sbbusy` until the bus is not busy?

Reply:
Yes. Fixing...